src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -255,8 +259,12 @@ public int compare(FeedbackQuestionAttributes q1, FeedbackQuestionAttributes q2)
         return getFeedbackQuestionsForCreatorInstructor(fsa);
     }
     
+    /**
+     * Gets a {@code List} of all questions for the list of questions that an
+     * instructor who is the creator of the course can view/submit
```

Comment:
The explanation is quite confusing. "... all questions ~~for the list of questions~~ that an ..." should be clearer.
Also, if the header is accurate, the method name might have to be improved. The word `Creator` can be ambiguous: session creator or course creator?